service/src/LevelZeroIOGroup.cpp

```diff
@@ -447,6 +447,38 @@ namespace geopm
                                   },
                                   .01
                                   }},
+                              {M_NAME_PREFIX + "GPU_CORE_TEMPERATURE_MAXIMUM", {
+                                  "The maximum measured temperature across all sensors in the GPU accelerator.",
+                                  GEOPM_DOMAIN_GPU_CHIP,
+                                  Agg::average,
```

Comment:
```suggestion
                                  Agg::max,
```
Does this matter in this case?
Perhaps if you issued ```geopmread GPU_CORE_TEMPERATURE_MAXIMUM board 0``` and you wanted to see the absolute maximum across all GPUs?  As it is, reading at the board domain would take the average of all the GPUs max and return that.